common-content/en/module/how-our-curriculum-works/workshops/index.md

```diff
@@ -0,0 +1,13 @@
++++
+title="Workshops"
+headless="true"
+time= 30
+[objectives]
+1="Locate the CYF Workshops repo"
+2="Describe how we use workshops during Saturday sessions at CYF"
+3="Relate the use of workshops to active learning"
++++
+
+We do at least one active learning [Workshops](https://github.com/CodeYourFuture/CYF-Workshops/) every class day. Workshops are group activities where the community discusses, solves and reflects on problems related to their current sprint. They are an opportunity for practice, critical thinking and dialogue.
```

Comment:
Similar to my point above, I wonder if we should have some kind of glossary of CYF terms? I understand we're using a commonly understood term with "sprint", but it's a different context.